h2o-r/h2o-package/R/gam.R

```diff
@@ -116,9 +117,12 @@
 #' @param max_runtime_secs Maximum allowed runtime in seconds for model training. Use 0 to disable. Defaults to 0.
 #' @param custom_metric_func Reference to custom evaluation function, format: `language:keyName=funcName`
 #' @param num_knots Number of knots for gam predictors
-#' @param knot_ids String arrays storing frame keys of knots.  One for each gam column specified in gam_columns
-#' @param bs Basis function type for each gam predictors, 0 for cr
-#' @param scale Smoothing parameter for gam predictors
+#' @param knot_ids String arrays storing frame keys of knots.  One for each gam column set specified in gam_columns
+#' @param standardize_tp_gam_cols \code{Logical}. standardize tp (thin plate) predictor columns Defaults to FALSE.
```

Comment:
Minor updates: `Standardize tp (thin plate) predictor columns.  Defaults to FALSE.`